plugin/auto/README.md

```diff
@@ -76,3 +75,8 @@ org {
     }
 }
 ~~~
+
+## Also
```

Comment:
Minor nit, I'm not sure "Also" is a good section header.

Reply:
fixed